src/vs/editor/browser/widget/diffEditorWidget.ts

```diff
@@ -61,7 +61,7 @@ interface IEditorsZones {
 }
 
 interface IDiffEditorWidgetStyle {
-	getEditorsDiffDecorations(lineChanges: editorCommon.ILineChange[], ignoreTrimWhitespace: boolean, renderIndicators: boolean, originalWhitespaces: IEditorWhitespace[], modifiedWhitespaces: IEditorWhitespace[], originalEditor: editorBrowser.ICodeEditor, modifiedEditor: editorBrowser.ICodeEditor): IEditorsDiffDecorationsWithZones;
+	getEditorsDiffDecorations(lineChanges: editorCommon.ILineChange[], ignoreTrimWhitespace: boolean, renderIndicators: boolean, originalWhitespaces: IEditorWhitespace[], modifiedWhitespaces: IEditorWhitespace[], originalEditor: editorBrowser.ICodeEditor, modifiedEditor: editorBrowser.ICodeEditor, reverse?: boolean): IEditorsDiffDecorationsWithZones;
```

Comment:
Please add a `// {{SQL CARBON EDIT}} comment to each distinct block of code changed under the `vs` directory.  This is needed to help when merging in VS Code source code refreshes so we can easily identify our patches.

Also, can you please try to refactor this change to minimize the number of lines of code changed and distinct code blocks added under `vs` directory.  Perhaps by creating a help class under `sql` and then just adding method calls from under `vs`.  

Basically we want to keep changes to vs source code minimal and narrowly scoped.

Reply:
I moved some of the code to a new class diffEditorHelper.ts in sql/editor/browser. Is this the right place to put it?